pkg/custom_detectors/custom_detectors.go

```diff
@@ -82,66 +83,98 @@ func (c *customRegexWebhook) FromData(ctx context.Context, verify bool, data []b
 	// ]
 	matches := permutateMatches(regexMatches)
 
+	g := new(errgroup.Group)
```

Comment:
question: Are we okay not setting an upper bound on the number of goroutines here? Is the idea the len of matches will be low enough for this not to cause issues?

If we did plan on bounding the number of goroutines we can add:
`g.SetLimit(<some-limit>)`

Reply:
@mcastorina thoughts?

Reply:
That's what I was thinking. It's at most 100 goroutines that will be doing "async I/O" work.